### DISCUSSION TITLE
Evidence půjčeného vybavení v interní sekci

### DISCUSSION THREAD

**Discussion by TomasNeumann** [bosancz/bosan.cz#582] [Ideas]
Aktuálně zavádíme nový systém přístupu k erárnímu vodáckému vybavení
- vesty, pádla atp budeme půjčovat za kauci na dlouho dobu
- tzn děti se budou o svěřený materiál přes celý rok starat sami - budou ho mít ve skřínce (prozatím komplexy), budou si ho moct brát i na mimoskupinové akce atp
-> vzniká tedy potřeba kvalitní a co nejvíce user-friendly evidence

Jaké jsou možnosti (velmi hrubý nástřel)
- nejpraktičtější by mi přišlo vše komplet nacpat do interní sekce
- tzn vytvořit databáze vybavení pro každý typ věci
- u každého člověka pak bude možnost vybrat nemá/má půjčeno -> nezaplatil/zaplatil kauci -> kdy a komu (v poznámce)

**Comment by TomasNeumann** [comment]
Moc nevím jak by bylo náročné vše implementovat (a kolik by to stálo práce samozřejmě). To nechám ať se vyjádří Kopec :)

**Comment by SmallhillCZ** [comment]
@TomasNeumann: Jaká je výhoda to mít na webu vs Google Drive?

Naprogramování do interní má podle mě smysl tam, kde nám chybí funkcionality, které GDrive nenabízí (nebo blbě), nebo kde se to nějak napojuje na další věci (vedení akcí, kde si třeba vygeneruješ údaje o dětech z databáze). Tady to zatím nevidím, je to osamocený seznam.

**Comment by Grejdy** [comment]
No, aktuálně to na drivu funguje, ale je to takové těžkopádné… funguje to pomalu, člověk (pokud vím) tam při půjčování prohledává kilometrový seznam, aby našel správný kus, je to náchylné na rozbití (už se tam párkrát stalo, že někdo něco rozbil, aniž by o tom věděl), najít podle toho číslo ztraceného pádla (tedy… uklizeného do eráru) je taky bolest… mít to v interní sekci by bylo elegantnější.
